src/components/Pagination.tsx

```diff
@@ -1,11 +1,12 @@
-import React from 'react';
+import React, {FC} from 'react';
 import {useState, useEffect} from "react";
 import cn from 'classnames';
 
 import './Pagination.css';
-import {PaginationProps} from '../interface/Interface'
+import {PaginationProps} from '../interfaces/Interface'
 
-function Pagination({activePage, totalItems, perPage, withActions, classes, onChangePage}: PaginationProps) {
+
+const Pagination= ({activePage, totalItems, perPage, withActions, classes, onChangePage}: PaginationProps)=> {
```

Comment:
NIT: It`s not change of this PR, but it should be fixed:
Below there is a hook const [Page, SetPage] = useState(activePage); 
According to Bohdan`s guide (https://basarat.gitbook.io/typescript/styleguide#interface), we should use camel case for variables, so it would be good to rename Page to page and SetPage to setPage